kml/serdes_gpx.hpp

```diff
@@ -57,14 +58,19 @@ class GpxParser
   LocalizableString m_name;
   LocalizableString m_description;
   PredefinedColor m_predefinedColor;
-  m2::PointD m_org;
+  geometry::PointWithAltitude m_org;
 
   double m_lat;
   double m_lon;
+  int16_t m_altitude;
 
   MultiGeometry::LineT m_line;
   LocalizableString m_customName;
   std::vector<TrackLayer> m_trackLayers;
+  void ParseName(std::string const & value, std::string_view  prevTag);
```

Comment:
```suggestion
  void ParseName(std::string const & value, std::string const & prevTag);
```

Reply:
We now use std::string in m_tags, right?

Reply:
Fixed